x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/hooks.tsx

```diff
@@ -111,6 +114,11 @@ export const getActionStatus = (status: ResponseActionStatus): string => {
   return '';
 };
 
+export const getCommand = (
+  command: ResponseActions
+): Exclude<ResponseActions, 'unisolate' | 'running-processes'> | 'release' | 'processes' =>
+  command === 'unisolate' ? 'release' : command === 'running-processes' ? 'processes' : command;
```

Comment:
nit: I find nested ternaries are a bit hard to read. it'd be more lines but I think it would be easier to read if we just used `if`s or a `switch`.

Reply:
Agree. Will update in a new PR